site/ja/tutorials/keras/save_and_restore_models.ipynb

```diff
@@ -0,0 +1,850 @@
+        "# 重みとオプティマイザを含む全く同じモデルを再作成\n",
+        "new_model = keras.models.load_model('my_model.h5')\n",
+        "new_model.summary()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "JXQpbTicTBwt"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "正解率を検査します。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "jwEaj9DnTCVA",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "loss, acc = new_model.evaluate(test_images, test_labels)\n",
+        "print(\"Restored model, accuracy: {:5.2f}%\".format(100*acc))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "dGXqd4wWJl8O"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "この方法では、次のすべてが保存されます。\n",
+        "\n",
+        "* 重みの値\n",
+        "* モデルの設定（アーキテクチャ）\n",
+        "* オプティマイザの設定\n",
+        "\n",
+        "Kerasは保存する際にアーキテクチャを調べます。いまのところ、TensorFlowのオプティマイザ（`tf.train`に含まれるもの）を保存することはできません。TensorFlowのオプティマイザを使用している場合には、モデルをロードしたあと再コンパイルする必要があり、オプティマイザの状態は失われます。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "kPyhgcoVzqUB"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### `saved_model`として"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "LtcN4VIb7JkK"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "注意：この手法による`tf.keras`モデルの保存は実験的なもので、将来のバージョンで変更される可能性があります。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "DSWiSB0Q8c46"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "新しいモデルを作ります。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "sI1YvCDFzpl3",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "model = create_model()\n",
+        "\n",
+        "model.fit(train_images, train_labels, epochs=5)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "iUvT_3qE8hV5"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "`saved_model`を作成します。"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "sq8fPglI1RWA",
+        "scrolled": true,
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "saved_model_path = tf.contrib.saved_model.save_keras_model(model, \"./saved_models\")"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "MjpmyPfh8-1n"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "セーブドモデル（saved models） はタイムスタンプ付きのディレクトリに保存されます。"
```

Comment:
Weak Suggestion:
I think we need not to translate SavedModel because It is a name of a format.

Reply:
Thanks.  It's difficult to choose Katakana and English word.